pipelines/keras.Sequential.MLPMultiClassClassifier.json

```diff
@@ -0,0 +1,31 @@
+{
+    "metadata": {
+        "name": "keras.Sequential.MLPMultiClassClassifier",
+        "data_type": "single_table",
+        "task_type": "classification"
+    },
+    "validation": {
+        "dataset": "iris",
+        "context": {}
+    },
+    "primitives": [
+        "mlprimitives.custom.counters.UniqueCounter",
+        "keras.Sequential.MLPMultiClassClassifier"
+    ],
+    "input_names": {
+        "mlprimitives.custom.counters.UniqueCounter#1": {
+            "X": "y"
+        }
+    },
+    "output_names": {
+        "mlprimitives.custom.counters.UniqueCounter#1": {
+            "counts": "classes"
+        }
+    },
+    "init_params": {
+        "keras.Sequential.MLPMultiClassClassifier#1": {
+            "epochs": 1,
```

Comment:
Since this example is very quick, let's remove this and let the default (20) be used.